### PR TITLE
Refactor `cleaning.py` script (empty branch)

### PR DESCRIPTION
**Pull request by 2010b9**
The cleaning.py script was refactored to include 3 main functions: load_data(), clean_data() and save_data().

Relates to https://github.com/2010b9/learning-pod-foundations-assignments/pull/4

### CODE REVIEW

File: life_expectancy/cleaning.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+from pathlib import Path
+import argparse
+import pandas as pd
+
+
+CURRENT_DIR = str(Path(__file__).parent)
+LIFE_EXPECTANCY_DATA_PATH = CURRENT_DIR + "/data/eu_life_expectancy_raw.tsv"
+OUTPUT_DATA_PATH = CURRENT_DIR + "/data/pt_life_expectancy.csv"
```
Comment (erstome) [inline life_expectancy/cleaning.py:8]: If you filter for a region that is not 'PT', the output will still be saved as 'pt_life_expectancy.csv". I would suggest making the name of the outputted *.csv file dynamic (in accordance with the selected region).

File: life_expectancy/cleaning.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    return pd.read_table(LIFE_EXPECTANCY_DATA_PATH)
+
+
+def clean_data(
+    life_expectancy_df: pd.DataFrame, region: str = "PT"
+) -> pd.DataFrame:
+    """Cleans the data in the life expectancy DataFrame"""
+    return (
+        life_expectancy_df
+        .pipe(_split_column_into_several)
```
Comment (erstome) [inline life_expectancy/cleaning.py:29]: Very nice the dataframe.pipe() method! I didn't know about it. I think I will use it in my code too.

File: life_expectancy/cleaning.py (hunk 1 above)
Comment (2010b9) [reply life_expectancy/cleaning.py]: Good catch! I solved it in this commit https://github.com/2010b9/learning-pod-foundations-assignments/pull/5/commits/e40b93399e9878f75159fdc36b485cdf57958427 🙂

File: life_expectancy/cleaning.py (hunk 2 above)
Comment (2010b9) [reply life_expectancy/cleaning.py:28]: I found out about it a few weeks ago! It's really cool to create a DataFrame pipeline

**Pull request merged**